src/RuntimeLoader/FactoryRuntimeLoader.php

```diff
@@ -18,7 +18,10 @@
  */
 class FactoryRuntimeLoader implements RuntimeLoaderInterface
 {
-    private $map;
+    /**
+     * @var callable[]
```

Comment:
`array<string, callable(): object>` (and same should be added on the constructor)

Reply:
Done